docs/api.md

```diff
@@ -38,10 +38,13 @@ These are the CRDs currently available in `crd.antrea.io`.
 | `ExternalEntity` | v1alpha2 | v1.0.0 | N/A | N/A |
 | `ExternalIPPool` | v1alpha2 | v1.2.0 | v1.13.0 | N/A |
 | `ExternalIPPool` | v1beta1 | v1.13.0 | N/A | N/A |
+| `ExternalNode`   | v1alpha1 | v1.8.0 | N/A | N/A |
+| `IPPool`| v1alpha2 | v1.4.0 | N/A | N/A |
 | `Group` | v1alpha3 | v1.8.0 | v1.13.0 | N/A |
 | `Group` | v1beta1 | v1.13.0 | N/A | N/A |
 | `NetworkPolicy` | v1alpha1 | v1.0.0 | v1.13.0 | N/A |
 | `NetworkPolicy` | v1beta1 | v1.13.0 | N/A | N/A |
+| `SupportBundleCollection` | v1alpha1 | v1.9.0 | N/A | N/A |
```

Comment:
This is 1.10.0

Reply:
I was checking this commit and the oldest version is v1.9.0 https://github.com/antrea-io/antrea/commit/5dfdf248e6e09e75163060dafefab3d9688d2a2a.
Do you mean the feature is officially introduced in 1.10.0?

Reply:
I just checked that the version info in the antrea_features.go is v1.10. I will update it.

Reply:
Fixed.

Reply:
All of our documents (changelog, feature doc, feature gate) point to v1.10.0, so we should be consistent. 1.9.0 may be the version the definition was added but guess it was not implemented by then.